srcpkgs/ibus-array/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'ibus-array'
+pkgname=ibus-array
+version=0.2.2
+revision=1
+#archs="i686 x86_64"
+wrksrc=${pkgname}-release-${version}
+#create_wrksrc=yes
+build_style=gnu-configure
+configure_args="prefix=/usr"
+#make_build_args=""
+#make_install_args=""
+#conf_files=""
+#make_dirs="/var/log/dir 0755 root root"
+hostmakedepends="automake pkg-config python ibus-devel gettext-devel sqlite-devel libtool"
+makedepends="ibus-devel gettext-devel sqlite-devel"
+depends="ibus"
+short_desc="Array 30 engine for ibus"
+maintainer="CToID <funk443@yandex.com>"
+license="GPL-2.0-only"
+homepage="https://github.com/lexical/ibus-array"
+#changelog=""
+distfiles="https://github.com/lexical/ibus-array/archive/refs/tags/release-${version}.tar.gz"
+checksum=736fee6de90674619f6ad9b0821dc989d91c7b9f364cd7b2b228fdecd9814f85
+python_version=3.10.4
+
+pre_configure() {
+	$wrksrc/autogen.sh --host=${os}
```

Comment:
could you verify locally that cross build work?

Reply:
It seems to build correctly now.